src/status_im2/contexts/communities/overview/style.cljs

```diff
@@ -33,3 +33,9 @@
    :margin-top   12
    :margin-left  :auto
    :margin-right :auto})
+
+(def community-overview-container
+  {:position :absolute
+   :top      (if platform/ios? 0 44)
+   :width    "100%"
+   :height   "110%"})
```

Comment:
we should never use % for dimensions, this looks like magic

Reply:
This is a part of the code that I didn't write, just moved. But I'm not sure why in general we shouldn't use percentages for dimensions? `:width "100%"` looks quite self-explanatory to me.

Could you please elaborate a bit? Maybe we can put the reason into the guidelines document

Reply:
I believe he is referring to height 110%. This is needed as the screen needs to overflow somewhat and tbh it doesn't matter how much it overflows, just more than the 100%

Reply:
Perhaps he's referring to the fact that percentages are rarely needed in a flexbox rendering model, which is the case of RN.

Reply:
so here two different issues
1) we never should use percentage
  a) because 100% doesn't make sense in flexbox
   b) because we always have fixed margins or paddings in the design, so instead of useing 80% we  should use padding-horizontal 20, for example, because % will be always different on differrent devices, but we should always have same paddings in px on all devices
2) 110% and top looks like a hack, hacks are bad, better to find a proper reliable solution